srcpkgs/pdf2djvu/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdf2djvu'
+pkgname=pdf2djvu
+version=0.9.19
+revision=1
+#archs="i686 x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=gnu-configure
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="pkg-config"
+makedepends="djvulibre djvulibre-devel libdjvulibre poppler poppler-devel GraphicsMagick libgraphicsmagick libgraphicsmagick-devel exiv2 exiv2-devel libuuid libuuid-devel gettext gettext-devel"
+depends="djvulibre libdjvulibre poppler GraphicsMagick libgraphicsmagick exiv2 libuuid gettext"
```

Comment:
all of the required dependencies should be autodetected, remove this line

Reply:
Thanks, fixed